omnibus/files/private-chef-cookbooks/private-chef/recipes/rabbitmq.rb

```diff
@@ -74,13 +75,37 @@
   variables(rabbitmq.to_hash)
 end
 
+#ssl_keyfile = File.join(rabbitmq_ca_dir, "rabbitmq.key")
+ssl_crtfile = File.join(rabbitmq_ca_dir, "rabbitmq.crt")
+
+openssl_x509 ssl_crtfile do
+  common_name node['private_chef']['rabbitmq']['server_name']
+  org node['private_chef']['rabbitmq']['ssl_company_name']
+  org_unit node['private_chef']['rabbitmq']['ssl_organizational_unit_name']
+  country node['private_chef']['rabbitmq']['ssl_country_name']
+  key_length node['private_chef']['rabbitmq']['ssl_key_length']
+  expire node['private_chef']['rabbitmq']['ssl_duration']
+  owner 'root'
+  group 'root'
+  mode '0644'
+end
+
+template "#{rabbitmq_etc_dir}/rabbitmq.config" do
+  owner "root"
+  group "root"
+  mode "0755"
+  variables( :rabbitmq_ca_dir => rabbitmq_ca_dir )
+end
+
 component_runit_service "rabbitmq"
 
 if is_data_master?
   rmq_ctl = "/opt/opscode/embedded/bin/rabbitmqctl"
+  rmq_plugins = "/opt/opscode/embedded/bin/rabbitmq-plugins"
   opc_ctl = "/opt/opscode/bin/private-chef-ctl"
   opc_username = OmnibusHelper.new(node).ownership['owner']
   rmq_ctl_chpost = "/opt/opscode/embedded/bin/chpst -u #{opc_username} -U #{opc_username} #{rmq_ctl}"
+  rmq_plugins_chpost = "/opt/opscode/embedded/bin/chpst -u #{opc_username} -U #{opc_username} #{rmq_plugins}"
```

Comment:
I see that the `_chpost` extension is used above, but part of me feels that this is a typo for `_chpst` that has lived for a while.